backend/src/main/java/ca/bc/gov/backendstartapi/entity/ActiveOrchardSeedPlanningUnit.java

```diff
@@ -0,0 +1,62 @@
+package ca.bc.gov.backendstartapi.entity;
+
+import ca.bc.gov.backendstartapi.entity.idclass.ActiveOrchardSeedPlanningUnitId;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.IdClass;
+import jakarta.persistence.Table;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+/** Auxiliary entity connecting an Orchard and a Seed Plan Unit (SPU). */
+@Entity
+@Table(name = "active_orchard_spu")
+@IdClass(ActiveOrchardSeedPlanningUnitId.class)
+@NoArgsConstructor(access = AccessLevel.PACKAGE, force = true)
+@RequiredArgsConstructor
+@Getter
+@Setter
+@Schema(description = "An association between an orchard and a Seed Plan Unit (SPU).")
+public class ActiveOrchardSeedPlanningUnit {
+
+  @Id
+  @Column(name = "orchard_id", length = 3, nullable = false)
```

Comment:
You can dismiss `nullable = false` here, because `nullable = false` is used for generating the table schema or doing validations when reading from the database. However you're creating the schema with proper constraints, instead of Hibernate, and you annotated with `@NotNull`, preventing to save null values. Same for `seedPlanningUnitId`.